packages/blade/src/components/Typography/Code/Code.tsx

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+import { BaseText } from '../BaseText';
+import Box from '~components/Box';
+import type { TextTypes } from '~tokens/theme/theme';
+import { getPlatformType } from '~utils';
+
+export type CodeProps = {
+  /**
+   * children only supports string.
+   *
+   * ```tsx
+   * // ❌ Bad
+   * <Code><Box>something</Box></Code>
+   * ```
+   *
+   *```tsx
+   * // ✅ Good
+   * <Code>something</Code>
+   * ```
```

Comment:
Don't think this is needed, since the TS error will be there anyway.

Reply:
done